core/src/traits/Kokkos_OccupancyControlTrait.hpp

```diff
@@ -62,11 +62,15 @@ struct MaximizeOccupancy;
 
 struct DesiredOccupancy {
   int m_occ = 100;
-  explicit constexpr DesiredOccupancy(int occ) : m_occ(occ) {
+  bool tune;
+  explicit constexpr DesiredOccupancy(int occ) : m_occ(occ), tune(false) {
     KOKKOS_EXPECTS(0 <= occ && occ <= 100);
   }
+  explicit constexpr DesiredOccupancy(const Kokkos::AUTO_t)
+      : m_occ(100), tune(true) {}
```

Comment:
There is no need to set `m_occ` it is done line 64

Reply:
Thanks, 
051f40d